src/index.html

```diff
@@ -11,7 +11,159 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="frame" name="personalInformation">
+        <legend>Personal information</legend>
+        <label for="surname" class="item">Surname:</label>
+        <input
+        type="name"
+        id="surname"
+        required
+        class="item"
+        name="surname"
+        ><br>
+
+        <label for="name" class="item">Name:</label>
+        <input
+        type="name"
+        id="name"
+        class="item"
+        name="name"
+        required
+        ><br>
```

Comment:
fix indents for attributes, fix in all places
```suggestion
        <input
          type="name"
          id="name"
          class="item"
          name="name"
          required
        >
```